plots/quasirandom.py

```diff
@@ -44,29 +43,29 @@ def f(x, y):
 def trapez(f, N_list):
     res = []
     for Ntgt in N_list:
-        N = int(ceil(Ntgt**0.5))
+        N = int(np.ceil(Ntgt**0.5))
         h = 2.0/N
-        pos = arange(0, N)*h + h/2 - 1.0
+        pos = np.arange(0, N)*h + h/2 - 1.0
         S = 0
         for y in pos:
             S += sum(f(pos, y))
 
         res.append(S*(2.0/N)**2)
 
-    return array(res)
+    return np.array(res)
 
 def mc(f, N_list):
     res = []
     for N in N_list:
-        res.append(2.0**2*sum(f(uniform(-1,1,N), uniform(-1,1,N)))/N)
-    return array(res)
+        res.append(2.0**2*sum(f(np.random.uniform(-1,1,N), np.random.uniform(-1,1,N)))/N)
+    return np.array(res)
 
 def qmc(f, N_list):
     res = []
```

Comment:
Das sollte wohl eine List-Comprehension sein, res = [2.0**2*sum(...) for N in N_list]